File: src/main/java/ru/mdashlw/hypixel/ranked/reply/impl/GameReply.java
package ru.mdashlw.hypixel.ranked.reply.impl;

import com.fasterxml.jackson.annotation.JsonProperty;
import ru.mdashlw.hypixel.ranked.entities.Game;
import ru.mdashlw.hypixel.ranked.reply.Reply;

public final class GameReply extends Reply<Game> {
    @JsonProperty
    private Game game;

    @Override
    public Game getValue() {
        return this.game;
    }
}

File: src/main/kotlin/ru/mdashlw/hypixel/api/ranked/reply/impl/GameReply.kt
package ru.mdashlw.hypixel.api.ranked.reply.impl

import ru.mdashlw.hypixel.api.ranked.entities.Game
import ru.mdashlw.hypixel.api.ranked.reply.Reply

class GameReply(
    success: Boolean,
    cause: String?,
    game: Game?
) : Reply<Game>(success, cause, game)
